runtime/browser/runtime.cc

```diff
@@ -342,4 +342,18 @@ void Runtime::LoadProgressChanged(content::WebContents* source,
     ui_delegate_->SetLoadProgress(progress);
 }
 
+void Runtime::AddDownloadItem(content::DownloadItem* download_item,
+    const content::DownloadTargetCallback& callback,
+    const base::FilePath& suggested_path) {
+  if (ui_delegate_) {
+    ui_delegate_->AddDownloadItem(download_item, callback, suggested_path);
+  } else {
+    const base::FilePath empty;
+    callback.Run(empty,
```

Comment:
I mean why not simply:
{
if (ui_delegate_) 
    ui_delegate_->AddDownloadItem(download_item, callback, suggested_path);
}